datafusion/physical-expr/src/expressions/binary.rs

```diff
@@ -1080,7 +1071,7 @@ mod tests {
             Operator::Plus,
             Int32Array,
             DataType::Int32,
-            vec![2i32, 4i32]
+            vec![2i32, 4i32],
         );
         test_coercion!(
```

Comment:
I wonder if these tests are doing anything useful anymore now that we are coercing in the logical layer? 🤔 It seems like they are now testing the test code 😆